tests/lent/tbasic_lent_check.nim

```diff
@@ -1,4 +1,5 @@
 discard """
+  targets: "c cpp"
```

Comment:
js?

Reply:
I am not fixing VM or JS in this PR

Reply:
yes but js and already work, so it would make sense to increase test coverage by including those as regression tests. Either way, I can do a followup PR add those.

Reply:
=> fixing this in https://github.com/nim-lang/Nim/pull/16075